### DISCUSSION TITLE
Using DigitalOcean Kube

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#893] [Ideas]
https://www.digitalocean.com/docs/kubernetes/quickstart/

1. Create a cluster
1. Install kubectl, doctl
1. Config file command
1. Deploy nginx ingress load balancer
1. Deploy sample app

Question
- cost for multiple apps?

**Comment by naomi-lgbt** [comment]
Step 1: Do magic
Step 2: Get app

**Comment by eddiejaoude** [comment]
I got it to work with a different docker container `nginx` - doing the exact same thing as we did without `ingress` - some more configuration must be needed for the `mongo-express` container to work

**Reply by naomi-lgbt** [reply]
Did magic, got app. 👍🏻

**Reply by eddiejaoude** [reply]
Made a video on it. 

Hopefully will learn more and do a part 2 🤓
